packages/launch/src/hooks/use-on-launch.ts

```diff
@@ -72,5 +62,5 @@ export const useOnLaunch = () => {
 			}
 			window.top.location.href = `https://wordpress.com/home/${ siteId }`;
```

Comment:
I know it's not in the scope of this PR, but it'd be nice if we could the `redirectTo` function from the Launch context, instead of hardcoding the redirection.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/47601/commits/d20436594404d073ad4c931bf67f9b92cfb067f6